release.sh

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
I'm inclined to say this PR will hurt more than help: http://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang